athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandler.java

```diff
@@ -111,55 +95,22 @@ protected NeptuneRecordHandler(final AmazonS3 amazonS3, final AWSSecretsManager
      *       performance.
      */
     @Override
-    protected void readWithConstraint(final BlockSpiller spiller, final ReadRecordsRequest recordsRequest,
-            final QueryStatusChecker queryStatusChecker) throws Exception 
+    protected void readWithConstraint(final BlockSpiller spiller, final ReadRecordsRequest recordsRequest, final QueryStatusChecker queryStatusChecker) throws Exception 
```

Comment:
this line is too long, please restore the previous wrapping to two lines.

Reply:
reverted